Sources/PackLib/Planner.swift

```diff
@@ -153,7 +153,8 @@ public struct Planner: Sendable {
             bundleID: bundleID,
             infoPlist: infoPlist,
             resources: resources,
-            iconPath: self.schema.base.iconPath
+            iconPath: self.schema.base.iconPath,
+            entitlementsPath: self.schema.base.entitlementsPath,
```

Comment:
this trailing comma breaks Swift 6.0 support, worth it

Reply:
oh… build.yml defaults to Xcode 16.2 / Swift 6.0 so I suppose we're removing the comma for now 🙃

Reply:
could bump to Xcode 16.3 but tons of people are still on 16.2 afaict